cli/config.go

```diff
@@ -125,6 +126,7 @@ func registerFlags(c *Config, f *flag.FlagSet) {
 	f.StringVar(&c.CNOverride, "cn", "", "certificate common name (CN)")
 	f.StringVar(&c.AKI, "aki", "", "certificate issuer (authority) key identifier")
 	f.StringVar(&c.DBConfigFile, "db-config", "", "certificate db configuration file")
+	f.StringVar(&c.Expiration, "expiry", "0", "time from now after which the certificate will expire (default: one week)")
```

Comment:
Which certificate is expiring? If this is for general certificates, one week is too short. If this is for the CRL, it should be noted that it's for the CRL.